build-tests/heft-action-plugin-test/README.md

```diff
@@ -0,0 +1,6 @@
+# heft-action-plugin-test
+
+This project folder is one of the **build-tests** for the [Heft](https://www.npmjs.com/package/@rushstack/heft)
+build system. This project exercises a custom Heft action.
+
+Please see the [Heft documentation](http://localhost:4000/pages/heft/overview/) for documentation and tutorials.
```

Comment:
This URL is incorrect. If it was copy+pasted from somewhere, we should fix it everywhere.